src/styles/components/header.scss

```diff
@@ -0,0 +1,28 @@
+.header {
+    width: 100%;
+
+    &__up {
+        @include flex-center;
+
+        position: relative;
+        background-color: steelblue;
+        height: 7.5rem;
```

Comment:
this point is to discuss :)

Reply:
as discussed it is a proper time to set up 62.% of font-size in `html` slector in your general file